Signup Front End

pmarks98 opened the pull request:
Worked on the frontend design of three views: Signup, Login, Splashscreen. 

All three views do not have navigation yet between them. 

I had to go down into some of the lower-level frameworks to get the keyboard hiding working on the signup. SwiftUI has no built-in feature to prevent the keyboard from displaying over the TextField inputs. 

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-10 at 10 08 27](https://user-images.githubusercontent.com/9061623/74176868-6a3df880-4bed-11ea-8e39-b6de68fffc0a.png)
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-10 at 10 08 45](https://user-images.githubusercontent.com/9061623/74176957-9ce7f100-4bed-11ea-8cfa-67f785dba03f.png)

![Simulator Screen Shot - iPhone 11 Pro Max - 2020-02-10 at 10 09 01](https://user-images.githubusercontent.com/9061623/74176888-732eca00-4bed-11ea-987c-01256f9ff074.png)

mrachamallu commented on frontend/RaffleBay/RaffleBay/View/SignupView.swift:
```diff
@@ -0,0 +1,225 @@
+    
+    //I don't know if this is the best/correct way to accept input.
+    //Another way I think could work is like a hashmap that maps inputs to their associated values. Will look into that.
+    @State private var firstName: String = ""
```
add these variables to the user.swift file in the model file. then make a user object in this class and update it's variables

mrachamallu commented on frontend/RaffleBay/RaffleBay/View/SignupView.swift:
```diff
@@ -0,0 +1,225 @@
+                        Text("Signup")
+                            .fontWeight(.medium)
+                            .padding(10)
+                            .font(.custom("Poppins", size: 30))
```
make a style.swift file that contains all the styling constants like "Poppins" etc

pmarks98 commented:
@mrachamallu I addressed your changes and cleaned up all the files. Made some changes to how the styles are applied and added many comments in the style.swift file for documentation on styling. I added in a constants.swift file and organized everything.

mrachamallu commented:
looks good. from what i've seen, the styles file is meant for constants only, and each different component, like a button, has it's own file. we can rearrange these things later in a different pr. feel free to merge @pmarks98
